README.md

```diff
@@ -204,7 +204,9 @@ Available shortcuts:
 | Name | Action | Options |
 | --- | --- | --- |
 | `grep_word_under_cursor` | Start live grep with word under cursor | <ul><li>`postfix`: postfix value to add; defaults to ` -F ` (Treat the pattern as a literal string)</li><li>`quote`: Whether to quote the value; defaults to true</li><li>`trim`: Whether to trim the value; defaults to true</li></ul> |
+| `grep_word_under_cursor_current_buffer` | Same as `grep_word_under_cursor` but for the file of the current buffer | |
 | `grep_visual_selection` | Start live grep with visual selection | see `grep_word_under_cursor` |
+| `grep_word_visual_selection_current_buffer` | Same as `grep_visual_selection` but for the file of the current buffer | |
```

Comment:
This is a little bit of confusing using `grep_word_visual_selection_current_buffer` as fucntion name.
It should be grep_visual_selection_current_buffer